### PR TITLE
[F] LRAC-14756 hide request buttons when the user has member permission

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LRAC-14756

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx [hunk 1]
```diff
@@ -315,6 +319,9 @@ const RequestList: React.FC<IRequestListProps> = ({
 		});
 	};
 
+	const isCurrentUserAsiteMember =
+		currentUser.roleName === UserRoleNames.Member;
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx:323]: Hey @fmarcio you can do it instead:

```suggestion
	const authorized = currentUser.isAdmin();
```

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx]: @interaminense

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @fmarcio

**Comment by interaminense** [comment]
ci:test:stable

**Comment by fmarcio** [comment]
ci:test:stable

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

**Comment by fmarcio** [comment]
@ViniLopes20 could you please check if the failure in the relevant test is related to this PR? thanks!

**Comment by ViniLopes20** [comment]
Hey @fmarcio, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

**Comment by fmarcio** [comment]
ci:forward:force

**Pull request closed without merging**